### PR TITLE
docs: update instructions for setting up ratify on Azure

### PR DESCRIPTION
**Pull request by shahramk64**
This PR aims to fix the issue #107 to resolve the problems with the instructions on setting up Ratify on Azure.

### CODE REVIEW

File: docs/quickstarts/ratify-on-azure.md [hunk 1]
```diff
@@ -276,13 +270,6 @@ az role assignment create --role "Key Vault Crypto User" --assignee $IDENTITY_OB
 
     This step above may take around 10 minutes to complete. The registration status can be checked by running the following command:
```
Comment (binbin-li) [inline docs/quickstarts/ratify-on-azure.md:271]:
```suggestion
    This step above may take around 10 minutes to complete.
```

File: docs/quickstarts/ratify-on-azure.md [hunk 2]
```diff
@@ -491,26 +478,25 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
...
-    name: verifier-notation
+      name: verifier-notation
     spec:
-    name: notation
-    artifactTypes: application/vnd.cncf.notary.signature
-    parameters:
+      name: notation
+      artifactTypes: application/vnd.cncf.notary.signature
+      parameters:
         verificationCertStores:
```
Comment (binbin-li) [inline docs/quickstarts/ratify-on-azure.md:486]: `verificationCertStores` is recommended to add the truststore type, like the `ca` in this case. https://ratify.dev/docs/next/plugins/verifier/notation#kubernetes

**Review by junczhu — COMMENTED** [review]
In the **Set up an Azure workload identity** shall we bring it up that using Azure Shell is also an option? 
Just wondering if you have experienced that having to manage all those exported values and a mix of different azure identities.

cc\ @yizha1 @FeynmanZhou

**Comment by junczhu** [comment]
> In the **Set up an Azure workload identity** shall we bring it up that using Azure Shell is also an option? Just wondering if you have experienced that having to manage all those exported values and a mix of different azure identities.
> 
> cc\ @yizha1 @FeynmanZhou

Found the Azure Cloud Shell was listed in the Prerequisites but just would prefer another call out if possible.

**Comment by junczhu** [comment]
> In the **Set up an Azure workload identity** shall we bring it up that using Azure Shell is also an option? Just wondering if you have experienced that having to manage all those exported values and a mix of different azure identities.
> 
> cc\ @yizha1 @FeynmanZhou

Found the Azure Cloud Shell was listed in the Prerequisites but just would prefer another call out if possible.

File: docs/quickstarts/ratify-on-azure.md (hunk 2 above)
Comment (shahramk64) [reply docs/quickstarts/ratify-on-azure.md:486]: @binbin-li 
When I revert this change to include `ca`, it doesn't successfully apply the yaml file:

kubectl describe Verifier verifier-notation
```
Name:         verifier-notation
Namespace:    
Labels:       <none>
Annotations:  helm.sh/hook: pre-install,pre-upgrade
              helm.sh/hook-weight: 5
API Version:  config.ratify.deislabs.io/v1beta1
Kind:         Verifier
Metadata:
  Creation Timestamp:  2024-08-27T00:06:43Z
  Generation:          2
  Resource Version:    1874082
  UID:                 13626af0-98f6-4de3-9ae2-99a1907a58c1
Spec:
  Artifact Types:  application/vnd.cncf.notary.signature
  Name:            notation
  Parameters:
    Trust Policy Doc:
      Trust Policies:
        Name:  default
        Registry Scopes:
          *
        Signature Verification:
          Level:  strict
        Trust Stores:
          ca:certs
        Trusted Identities:
          x509.subject: CN=wabbit-networks.io,O=Notation,L=Seattle,ST=WA,C=US
      Version:  1.0
    Verification Cert Stores:
      Ca:
        Certs:
          keymanagementprovider-akv
      Certs:
        ratify-notation-inline-cert-0
  Version:  1.0.0
Status:
  Brieferror:  Error: config invalid, Code: C...
  Error:       Error: config invalid, Code: CONFIG_INVALID, Plugin Name: verifier-notation, Component Type: verifier
  Issuccess:   false
Events:        <none>

```

I checked one sample notation config file here: ./conffig/samples/clustered/verifier/config-v1alpha1-verifier-notation.yaml and it also didn't include ca in verificationcertstores.

File: docs/quickstarts/ratify-on-azure.md [hunk 3]
```diff
@@ -389,13 +376,13 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
     apiVersion: config.ratify.deislabs.io/v1beta1
     kind: Store
     metadata:
-    name: store-oras
+      name: store-oras
```
Comment (susanshi) [inline docs/quickstarts/ratify-on-azure.md:379]: we probably want to revert these indentation

File: docs/quickstarts/ratify-on-azure.md (hunk 2 above)
Comment (binbin-li) [reply docs/quickstarts/ratify-on-azure.md:486]: curious if it should `ca` instead of `Ca`?

Comment (junczhu) [reply docs/quickstarts/ratify-on-azure.md:486]: No worries. I will answer this.

Comment (junczhu) [reply docs/quickstarts/ratify-on-azure.md:486]: You can retry after the https://github.com/ratify-project/ratify/pull/1758 get merged

**Review by susanshi — APPROVED** [review]
LGTM

**Pull request merged**